attestation/attestation.go

```diff
@@ -141,10 +140,52 @@ func CreateAIK(context *tspi.Context) ([]byte, []byte, error) {
 	return pubkey, blob, nil
 }
 
+// GetQuote consumes a nonce and the aik public blob and returns the quote,
+// a signature, and any error
+func GetQuote(context *tspi.Context, aikblob, nonce []byte) ([]byte, []byte, error) {
+	srk, err := context.LoadKeyByUUID(tspiconst.TSS_PS_TYPE_SYSTEM, tspi.TSS_UUID_SRK)
+	if err != nil {
+		return nil, nil, err
+	}
+	srkpolicy, err := srk.GetPolicy(tspiconst.TSS_POLICY_USAGE)
+	if err != nil {
+		return nil, nil, err
+	}
+	srkpolicy.SetSecret(tspiconst.TSS_SECRET_MODE_SHA1, wellKnown[:])
```

Comment:
Add a comment explaining why you're using wellKnown here. Could this ever be a different value?